src/styles/_variables.scss

```diff
@@ -1,10 +1,37 @@
 /*
+ * Base Settings
+ */
+
+$margin-vertical-sm: 1.666666667rem !default;
+$margin-vertical-lg: 5.555555556rem !default;
+
+$base-line-height:   1.555rem !default; // 28px
+$base-font-size:     18px !default;
+
+$gutter-width: 15px !default;
+
+$font-family-body:    "proxima-nova", sans-serif !default;
+$font-family-heading: "proxima-nova", sans-serif !default;
+$font-family-code:    "Consolas", "Monaco", "Andale Mono", "Ubuntu Mono", monospace !default;
+
+// Image Path.
+$images-path: "../images" !default;
```

Comment:
The main issue here was I couldn't figure out why the images were not being loaded on the new HM theme, the docs say to point `$hm-images` to the folder path in your project. Because `$hm-images` is no longer an alias of `$images-path`, this wasn't working anymore, but did work in V1.

I think they should all be prefixed with `hm-`, to be clear where the variable has come from. `$images-path:` is pretty generic and we wouldn't be able to use this in the main project we are using the pattern library in.

Perhaps we should consolidate and use `hm-` prefix internally in the project instead of aliasing?